controllers/lvm_volumegroup.go

```diff
@@ -113,3 +136,14 @@ func (c lvmVG) getLvmVolumeGroups(r *LVMClusterReconciler, instance *lvmv1alpha1
 	}
 	return lvmVolumeGroups
 }
+
+func doesVGExistOnHosts(ctx context.Context, volumeGroup string, instance *lvmv1alpha1.LVMCluster) (bool, error) {
```

Comment:
- it seems `error` is always nil and can be removed
- not a time limited func so `ctx` can be removed as well

Reply:
removed error.